docs/es/home.md

```diff
@@ -4,6 +4,8 @@ redirect_from: "/es"
 
```

Comment:
Here, put `# Home` but in Spanish ofc... https://www.azerothcore.org/wiki/wiki-standards#file-headers

Reply:
https://github.com/azerothcore/wiki/pull/599/commits/da9844c1d118001e689d466d5d971b62730cc573